src/Framework/ChangeWaves.cs

```diff
@@ -26,8 +26,8 @@ internal class ChangeWaves
     {
         internal static readonly Version Wave16_10 = new Version(16, 10);
         internal static readonly Version Wave17_0 = new Version(17, 0);
-        internal static readonly Version[] AllWaves = { Wave16_10, Wave17_0 };
-
+        internal static readonly Version Wave17_2 = new Version(17, 2);
```

Comment:
Do we want 17_2 stuff in here? Latest/vs17.2 doesn't have the wave, and if something gets merged into main now, the first LTS release that will see a new changewave feature is 17.4.

Reply:
I talked with marcpopMSFT about that a few days ago, and we decided yes. There isn't anything in it, but it avoids future confusion.